tiledb/sm/query/sparse_global_order_reader.cc

```diff
@@ -825,15 +827,15 @@ Status SparseGlobalOrderReader::merge_result_cell_slabs(
   return Status::Ok();
 };
 
-Status SparseGlobalOrderReader::resize_output_buffers(
-    std::vector<ResultCellSlab>& copied) {
-  // Copy might truncate the result cell slab.
+Status SparseGlobalOrderReader::resize_output_buffers() {
   // Count number of elements actually copied.
   uint64_t cells_copied = 0;
-  for (auto rcs : copied) {
-    cells_copied += rcs.length_;
+  for (uint64_t i = 0; i < copy_end_.first - 1; i++) {
```

Comment:
Is `copy_end_.first - 1` correct or should it be `copy_end_.first`?

Reply:
This line is correct because of cells_copied += copy_end_.second; below the for loop.